samples/force-sensitive-resistor/FsrWithAdcSample.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using Iot.Device.Mcp3008;
+
+namespace force_sensitive_resistor
+{
+    class FsrWithAdcSample
+    {
+        private int _pinNumber = 0; // pin number to which FSR output connected
```

Comment:
I'd remove the word `output` here since it depends on which side are you on (from perspective of the driver this will be input)